modules/mfa/public/prompt-for-mfa.php

```diff
@@ -32,6 +32,8 @@
 $cookieHash = filter_input(INPUT_COOKIE, 'c1') ?? ''; // hashed string
 $expireDate = filter_input(INPUT_COOKIE, 'c2') ?? 0;  // expiration timestamp
 if (Mfa::isRememberMeCookieValid(base64_decode($cookieHash), $expireDate, $mfaOptions, $state)) {
+    $idBrokerClient = Mfa::getIdBrokerClient($state['idBrokerConfig']);
+    $idBrokerClient->updateUserLastLogin($state['employeeId']);
```

Comment:
I think this sort of call should be happening in the AuthProc, not here in the view/controller file. Want to find a time to talk through this together?

Reply:
I'd actually thought about moving it back to mfa, I just hadn't gotten to it yet.